src/js/lib/cssVar.js

```diff
@@ -8,7 +8,7 @@ So we provide a fallback for the unit tests
 
 export function cssVar(name: string, fallback: string | number) {
   try {
-    return polished.cssVar(name)
+    return polishedCssVar(name)
```

Comment:
This was not loading in the right order. It now works when I import the named function instead of importing the default.